send_queue.go

```diff
@@ -2,13 +2,15 @@ package quic
 
 type sendQueue struct {
 	queue     chan *packedPacket
-	closeChan chan struct{}
+	closeChan chan struct{} // closed when Close() is called
+	closed    chan struct{} // closed when the run loop returns
```

Comment:
Nit: Maybe make the naming a bit more clear? E.g. `closeCalled` and `runStopped`?

Reply:
Done.